base/common/python/pki/crypto.py

```diff
@@ -194,6 +195,9 @@ def import_cert(self, cert_nick, cert, trust=',,'):
         else:
             content = cert
 
+        if sys.version_info[0] > 2:
+            content = content.encode()
```

Comment:
This patch fixes the issue I was seeing, however I'd suggest using six types instead, since you're already importing that module:

`if type(content) is not six.binary_type:
    content = content.encode()`

Reply:
Done! :+1: